src/screens/Connectors/ConnectorWalletDetails/Paze/PazeIntegrationUtils.res

```diff
@@ -0,0 +1,41 @@
+open PazeIntegrationTypes
+open LogicUtils
+
+let pazePayRequest = dict => {
+  client_id: dict->getString("client_id", ""),
+  client_name: dict->getString("client_name", ""),
+  client_profile_id: dict->getString("client_profile_id", ""),
+}
+
+let pazeNameMapper = (~name) => {
+  `connector_wallets_details.paze.${name}`
+}
+
+let pazeValueInput = (~pazeField: CommonConnectorTypes.inputField) => {
+  open CommonConnectorHelper
+  let {\"type", name} = pazeField
+  let formName = pazeNameMapper(~name)
```

Comment:
why make a func for a single line change

Reply:
yes later it can be extended